core/src/View/Kokkos_BasicView.hpp

```diff
@@ -264,12 +275,7 @@ class BasicView {
       !std::is_convertible_v<const OtherA &, accessor_type>)
 #endif
       KOKKOS_INLINE_FUNCTION
-      BasicView(const BasicView<OtherT, OtherE, OtherL, OtherA> &other,
-                std::enable_if_t<
-                    std::is_constructible_v<
-                        mdspan_type, typename BasicView<OtherT, OtherE, OtherL,
-                                                        OtherA>::mdspan_type>,
-                    void *> = nullptr)
+      BasicView(const BasicView<OtherT, OtherE, OtherL, OtherA> &other)
```

Comment:
The move here seems unrelated to the rest to me. Could you spell out why it is here?

Reply:
A lot of the stuff is unrelated to anything happening in this PR, because it's related to the new View implementation. But we explicitly broke the connection between related changes to have fewer changes.